src/core_modules/capture-core/components/DataEntry/DataEntry.component.js

```diff
@@ -65,7 +64,6 @@ const styles = theme => ({
         paddingBottom: theme.typography.pxToRem(10),
     },
     dataEntryFieldSection: {
-        padding: theme.typography.pxToRem(8),
```

Comment:
Removed this so the consumer of this component can choose how much padding the want it to have.